SleepEngine/Engine/Utils/stl_utils.h

```diff
@@ -141,4 +141,47 @@ TContainer filter(TContainer const& orig, TFilter filter)
 
 #pragma endregion
 
+#define SLP_ELEMENT_NOT_IN_MAP_MSG "Element not found in the map"
```

Comment:
Let's use auto const or const char* instead of this ugly define?

Reply:
won't it live in memory than?
and the main problem here is that it is a temp variable in header, anyway